src/components/search/SearchCourseCard.jsx

```diff
@@ -101,6 +102,7 @@ const SearchCourseCard = ({
     >
       <Card.ImageCap
         src={course.cardImageUrl || course.originalImageUrl}
```

Comment:
[consider] Should we also include `|| cardFallbackImg` in the `src` prop just in case `course.cardImageUrl` or `course.originalImageUrl` are null/undefined?

Reply:
Sounds like a good idea to me!